### DISCUSSION TITLE
[Explorer] Invocations for a given contract

### DISCUSSION THREAD

**Discussion by heytdep** [stellar/sorobanathon#88] [General]
This one also relies on https://github.com/xycloo/soroban-explorer like #84. It allows users to list and explore invocations of a given contract id. [Here's the website](contract-explore.xycloo.com), or [here if that one doesn't work](https://dist-soroban-operatons-per-contract-explore.pages.dev/), and the code is available [here](https://github.com/xycloo/soroban-explorer/tree/main/web/soroban-contract-ops-explore).

PS: I didn't know if to make a new submission or add it to #84, let me know if I should delete this post and add it there.

**Comment by achallagundla** [comment]
Heya @heytdep ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1000 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.
